cmd/context/create_test.go

```diff
@@ -292,7 +292,7 @@ func Test_createContext(t *testing.T) {
 			fakeOktetoClient := &client.FakeOktetoClient{
 				Namespace: client.NewFakeNamespaceClient([]types.Namespace{{ID: "test"}}, nil),
 				Users:     client.NewFakeUsersClient(tt.user),
-				Preview:   client.NewFakePreviewClient(nil, nil, nil),
+				Preview:   client.NewFakePreviewClient(&client.FakePreviewResponse{}),
```

Comment:
Can we get rid of the fake client on this test. It's not used by the function

Reply:
i've reviewd the ones that dont need the client